gendiff/scripts/gendiff.py

```diff
@@ -16,7 +17,49 @@ def main():
         help="set format of output",
     )
     args = parser.parse_args()
-    print(parser, args)
+    return args
+
+
+def generate_diff(first_file, second_file):
+    with (
+        open(first_file, "r") as file1,
+        open(second_file, "r") as file2,
+    ):
+        f1 = json.load(file1)
+        f2 = json.load(file2)
+        keys1 = list(f1.keys())
+        keys2 = list(f2.keys())
+        sorted_union = sorted(list(set(keys1 + keys2)))
+        diff = {}
+        for b in sorted_union:
+            if b in keys2 and b in keys1:
```

Comment:
is it possible to simplify this part? maybe, we could use some set operations to get missing, new, and changed keys